dotnet/src/SemanticKernel/Reliability/DefaultHttpRetryHandler.cs

```diff
@@ -123,15 +111,11 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 reason,
                 waitFor.TotalMilliseconds);
 
-            // Clone request with CloneAsync before retrying
-            // Do not dispose this request as that breaks the request cloning
-#pragma warning disable CA2000
-            request = await CloneAsync(request);
-#pragma warning restore CA2000
-
             // Increase retryCount
             retryCount++;
 
+            response?.Dispose();
```

Comment:
Confirm whether the response has a content there is issue re-firing the request. https://stackoverflow.com/questions/18000583/re-send-httprequestmessage-exception/18014515#18014515

Reach out to graph sdk folks or dotnet on why they do the cloning+draining.

Reply:
Confirmed with testing.

Communication started.